### DISCUSSION TITLE
Thoughts on swapping thumb Space and Backspace

### DISCUSSION THREAD

**Discussion by jasonish** [manna-harbour/miryoku#133] [Q&A] [answered]
I'm giving Miryoku a solid try to reduce movement of my right pinky.  I'm sticking to QWERTY for now, but may consider Colemak in the future.

Anyways, I have very strong muscle memory from a Kinesis Advantage where `backspace` is on the left thumb, and `space` on the right. Is there any thought out functionality or elegance that would be lost by making that change?

Thanks!

**Comment by manna-harbour** [comment, marked as answer]
Nav is for editing, so `Backspace` is important there. The best way, if you don't mind left hand Nav, would be the flip layers option https://github.com/manna-harbour/miryoku/tree/master/docs/reference#flip. Otherwise, you could swap `Backspace` and `Space` on Base and keep `Backspace` on Nav, but that might be confusing.

**Reply by jasonish** [reply]
Thanks. And you lose the repeating backspace on a layer.  Best to have the fingers relearn I think.  Thanks again.

**Reply by manna-harbour** [reply]
The `Backspace` on Nav will still auto repeat, it's just that the one on Base would be in a different position.

Also, you could disable https://github.com/qmk/qmk_firmware/blob/master/docs/tap_hold.md#tapping-force-hold for `Backspace` on Base to use auto repeat in the same position, but then you'd have to be careful to pause after tapping `Backspace` before holding for Nav.
